hathor/cli/events_simulator/events_simulator.py

```diff
@@ -80,7 +80,7 @@ def execute(args: Namespace) -> None:
 
     log.info('Started simulating events', scenario=args.scenario, seed=simulator.seed)
 
-    forwarding_ws_factory.start()
+    forwarding_ws_factory.start(stream_id='')
```

Comment:
Maybe call it `'simulator'`?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/766/commits/f10ea5624d35869490bbcc50a0ca1ceeb82f61bf

Reply:
Thanks!